core/iwasm/aot/aot_runtime.h

```diff
@@ -298,6 +312,8 @@ typedef struct AOTTargetInfo {
     uint32 e_version;
     /* Processor-specific flags */
     uint32 e_flags;
+    /* Specify wasm features supported */
+    uint32 feature_flags;
```

Comment:
Not sure whether uint32 is not enough for all features in the feature? Should we use uint64?

Reply:
Changed to uint64 now